dev/CommandBarFlyout/CommandBarFlyout.xaml

```diff
@@ -431,12 +433,6 @@
                                                 BorderBrush="{TemplateBinding BorderBrush}"
                                                 BorderThickness="{ThemeResource CommandBarFlyoutBorderThemeThickness}"
                                                 IsTabStop="False">
-                                                <CommandBarOverflowPresenter.ItemContainerStyle>
```

Comment:
Just wondering what happened with this? There are some elements that moved from having a width to being NaN.

Reply:
The `.ItemContainerStyle` bit here is overwriting and preventing the style template to be applied in `CommandBarFlyout_themeresources.xaml`. 

The `<Setter>` for width is being set to NaN to overwrite the CommandBar template it is based off of, and the actual default width is being set in the `ContentRoot` element through VisualTransitions. This is a workaround since FrameworkElement width is not settable via VisualTransitions, and it was preventing the style to be updated.

Reply:
Makes sense!